Themis/Views/Assessment/AssessmentSubmissionLoaderView.swift

```diff
@@ -27,6 +28,7 @@ struct AssessmentSubmissionLoaderView: View {
             if let pId = avm.submission?.participation.id {
                 await cvm.initFileTree(participationId: pId)
             }
+            UndoManagerSingleton.shared.undoManager.removeAllActions() /// to avoid the undo and redo of automatic feedbacks
```

Comment:
Instead of doing it here and onAppear and onDisappear and basically all around the code, just do it in AssessmentViewModel which has the only two methods that sets the initial feedbacks.